syncd/FlexCounter.cpp

```diff
@@ -479,6 +479,9 @@ class CounterContext : public BaseCounterContext
 {
     std::map<std::string, uint32_t> m_counterChunkSizeMapFromPrefix;
 
+protected:
+    sai_object_id_t m_switchId;
```

Comment:
this dont need to be a member, move it to function locally, since you are using this in 1 place

Reply:
but I do not need to fetch it every time it is called since the switch id isn't changed